config/prow/kustomization.yaml

```diff
@@ -4,6 +4,8 @@ resources:
 - base
 - external-plugins/cherrypicker_deployment.yaml
 - external-plugins/cherrypicker_service.yaml
+- external-plugins/needs-rebase_deployment.yaml
+- external-plugins/needs-rebase_service.yaml
```

Comment:
@richardcase any ideas why this is not applied to prow cluster once merged? I don't think it needs reorganization in customization but might be wrong..

Reply:
Because we don't have anything to run the kustomization